### PR TITLE
Removes Ununsed Code

### PR DESCRIPTION
**Pull request by KPrasch**
**Type of PR:**
Other

**Required reviews:** 
1

### CODE REVIEW

**Review by manumonti — APPROVED** [review]
LGTM! 👍

File: nucypher/cli/actions/select.py [hunk 1]
```diff
@@ -72,13 +67,6 @@ def select_client_account(
         testnet = domain != TACoDomain.MAINNET.name
```
Comment (derekpierre) [inline nucypher/cli/actions/select.py:67]: I know this is outside of the PR code, but we should be careful here if domain is `None`. If it is `None`, we assume testnet, is that desired?

File: tests/integration/cli/actions/test_select_client_account.py [hunk 2]
```diff
@@ -166,24 +177,21 @@ def test_select_client_account_with_balance_display(
...
-
     # Setup
     mock_staking_agent.get_all_stakes.return_value = stake_info
 
     # Missing network kwarg with balance display active
-    blockchain_read_required = any((show_staking, show_matic))
+    blockchain_read_required = any((show_matic,))
     if blockchain_read_required:
         with pytest.raises(
             ValueError, match="Pass domain name or registry; Got neither."
```
Comment (derekpierre) [inline tests/integration/cli/actions/test_select_client_account.py:190]: This part of the test is no longer applicable now...can this check be removed?

Comment (KPrasch) [reply tests/integration/cli/actions/test_select_client_account.py:190]: ✔️

File: nucypher/cli/actions/select.py (hunk 1 above)
Comment (KPrasch) [reply nucypher/cli/actions/select.py:67]: 🤔 I'll address that in #3315

Comment (KPrasch) [reply nucypher/cli/actions/select.py:67]: ✔️

Comment (derekpierre) [reply nucypher/cli/actions/select.py:67]: Was that commit pushed? Can't find it - https://github.com/nucypher/nucypher/commit/2ff02b23368b0ae84c2ad7d5cb1796ee00956b5f

Comment (KPrasch) [reply nucypher/cli/actions/select.py:67]: Try this one: 6d5ced138f28e39f9329592e39a2d02830f5928b

Comment (derekpierre) [reply nucypher/cli/actions/select.py:67]: Still not recognized as being part of any branch/pr... - 
Screenshot 2023-10-30 at 4 10 34 PM

Comment (KPrasch) [reply nucypher/cli/actions/select.py:67]: Sure, that's because it's in an unmerged PR -- is there some other way to link it?

Comment (derekpierre) [reply nucypher/cli/actions/select.py:67]: > 🤔 ~~I'll address that in https://github.com/nucypher/nucypher/pull/3315~~ handled in https://github.com/nucypher/nucypher/commit/2ff02b23368b0ae84c2ad7d5cb1796ee00956b5f

Oh no worries, I read your scratched off comment above as no longer being addressed in 3315, and assumed it was handled in this PR. All good then

**Pull request merged**